YACS-RCOS/yacs.n
# Fix ICS export & update metadata

**Issue**

Closes #125 

**Test Procedure**

Show procedure to test functionality with a clear procedure

1. Build & run the project. Upload Fall 2020 semester if not already there.
2. On schedule page, add class Data Mining `CSCI-4390`
3. Export to ICS (button is near the footer)
4. Look at in your favorite calendar app and one event for tues & fri should appear weekly @ 10am-11:50am until the week of 12/21.
5. The event information should show 'Graph Mining' as the class and the location should be blank.

**Photos**

N/A

**Additional Info**

Some clients care about capitalization of ICS rules. Problem was b/c `rrule` should have been capitalized. Haha.

(Credit to @jshom for finding that weird bug)
